plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -706,13 +706,11 @@ end
 
 function OPDSBrowser:browseSearchable(browse_url, username, password)
     self.search_server_dialog = InputDialog:new{
-        title = _("Search OPDS catalog"),
+        title = _("Enter author or title to search for"),
         input = "",
-        hint = _("Search string"),
         -- @translators: This is an input hint for something to search for in an OPDS catalog, namely a famous author everyone knows. It probably doesn't need to be localized, but this is just here in case another name or book title would be more appropriate outside of a European context.
         input_hint = _("Alexandre Dumas"),
         input_type = "string",
-        description = _("%s in url will be replaced by your input"),
```

Comment:
This strikes me as rather important information?
![Screenshot_2021-04-02_13-15-50](https://user-images.githubusercontent.com/202757/113411247-8f736480-93b5-11eb-8ef5-9dfc2f946c50.png)

Reply:
Assume I am not an IT man.
What is `%s`? In what url? I just want to search for Dan Brown.
And even if I understand the above, does it change my input?

Reply:
Hold on, let me check the git blame for this one. I'm not very much at home in the OPDS plugin. Indeed it seems somewhat random here. But the point remains that it's important information *somewhere*. If it's here rather than where it should be, that may be an accident that should be investigated rather than removed wholesale.

If you add a URL like `https://www.bing.com/search?FORM=INCOH2&PC=IFJ1&PTAG=ICO-c9d0fc87&q=%s` then `%s` will be the encoded search terms. Leave out the %s and you have no search.

Reply:
Context: https://github.com/koreader/koreader/pull/5823

Reply:
I remember it was redesigned. But...
https://github.com/koreader/koreader/pull/7380#issuecomment-792099693

Reply:
IIRC, it puzzled me, and I checked, and I *think* it still makes sense is some edge-cases w/ legacy searches.

But it's weird.

Reply:
So, basically, what @Frenzie said, it makes sense in like 0.5% of the callers of this dialog ;p.

Reply:
Thanks, have reverted already.